src/Avalonia.Controls/Grid.cs

```diff
@@ -706,28 +731,35 @@ private void ValidateCellsCore()
                     continue;
                 }
 
-                CellCache cell = new CellCache();
-
+                var cell = new CellCache();
+                var control = (Control)child;
 
                 //  Read indices from the corresponding properties:
                 //      clamp to value < number_of_columns
```

Comment:
This comment isn't really true any more is it? The value's no longer being clamped.